### PR TITLE
Chapter 14: Following users

### PR DESCRIPTION
**Pull request by NobiHust**
chap14

### CODE REVIEW

**Comment by NobiHust** [comment]
ready

File: app/models/relationship.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```
Comment (hungcnh-0828) [inline app/models/relationship.rb]:
```suggestion
  belongs_to :follower, class_name: User.name
```

File: app/models/user.rb [hunk 2]
```diff
@@ -1,5 +1,13 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name:  "Relationship",
```
Comment (hungcnh-0828) [inline app/models/user.rb]:
```suggestion
  has_many :active_relationships, class_name:  Relationship.name,
```

File: app/models/user.rb [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+                                  foreign_key: "follower_id",
```
Comment (hungcnh-0828) [inline app/models/user.rb]:
```suggestion
                                  foreign_key: :follower_id,
```

**Pull request merged**